app/src/main/res/menu/menu.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <item
+        android:id="@+id/back_button"
```

Comment:
So, if I understand right I have to create a toolbar item in fragment layout ```fragment_all_files.xml```? And make invisible the app bar?

Reply:
Because if I create a toolbar in fragment I have something like this 
![image](https://user-images.githubusercontent.com/55796198/133937665-3f1ccb9b-b953-42c0-9d97-455c932c6372.png)

Reply:
@gdragan I created back navigation using toolbar, check it please :)